emission/storage/timeseries/builtin_timeseries.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, user_id):
                 "manual/incident": self.timeseries_db,
                 "manual/mode_confirm": self.timeseries_db,
                 "manual/purpose_confirm": self.timeseries_db,
+                "manual/survey_response": self.timeseries_db,
```

Comment:
@atton16, it seems like @shankari's team is going with `survey_response`. Maybe it is what we should use as well for our Enketo plugin?

Reply:
There is a "manual/survey" in `gis-based-mode-detection` branch though. So ???

Reply:
But this one works for me anyways.